test/geometry/test_quaternion.py

```diff
@@ -221,3 +221,37 @@ def test_slerp(self, device, dtype, batch_size):
                 q3 = q1.slerp(q2, t)
                 q4 = Quaternion.from_axis_angle(axis * t * 3.14159)
                 self.assert_close(q3, q4)
+
+    def test_from_to_euler_values(self, device, dtype):
```

Comment:
not necessarily but add if you want some smoke values that could help to find out corner cases including  zeros or negative angles

Reply:
I've added the [0,0,0] test. There were already positive and negative angles in there. Hopefully that's what you meant ;)